docs/how-to-guides/CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+# Contributing to Facets Knowledge Panels
+
+First off, thanks for taking the time to contribute! 🎉🎉
+
+When contributing to this repository, please first discuss the change you wish to make via issue with the maintainers of this repository before making a change. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Open Food Facts Code of Conduct](https://world.openfoodfacts.org/code-of-conduct), please follow it in all your interaction with the project. By participating, you are expected to uphold this code. Please report unacceptable behavior/abuse to abuse@openfoodfacts.org
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build. Add only relevant files to commit and ignore the rest to keep the repo clean.
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
+3. You should request review from the maintainers once you submit the Pull Request.
+
+## Instructions
+
+- Git Workflow (all the commands are to run in your console)
+
+```bash
+## Step 1: Fork Repository
+
+## Step 2: Git Set Up & Download
+# Clone the repo
+git clone https://github.com/<User-Name>/facets-knowledge-panels.git
+
+# move to the folder
+cd facets-knowledge-panels
+
+# Add upstream remote
+git remote add upstream https://github.com/openfoodfacts/facets-knowledge-panels.git
+
+# Fetch and merge with upstream/master
+git fetch upstream
+git merge upstream/master
+
+## Step 2: Create and Publish Working Branch
+git checkout -b <type>/<issue-issueNumber>/{<additional-fixes>}
+
+# Example: fix/issue-21
+git push origin <type>/<issue-issueNumber>/{<additional-fixes>}
+
+## Types:
+# wip - Work in Progress; long term work; mainstream changes;
+# feat - New Feature; future planned; non-mainstream changes;
+# bug - Bug Fixes
+# exp - Experimental; random experimental features;
+```
+
+- On Task Completion:
+
```

Comment:
I would say we can put the link for "project setup" before On Task Completion, maybe the title can be  "setup project locally"